### PR TITLE
Added loggers and added credits

### PR DESCRIPTION
**Pull request by Nanosync**

### CODE REVIEW

File: src/main/java/seedu/moneygowhere/logic/commands/FindCommand.java [hunk 1]
```diff
@@ -37,6 +39,8 @@
 
     private final List<Predicate<Spending>> predicates;
 
+    private final Logger logger = Logger.getLogger("Find Command Logger");
```
Comment (jonathantjendana) [inline src/main/java/seedu/moneygowhere/logic/commands/FindCommand.java]: Should use Logger.getLogger(FindCommand.class) instead

File: src/main/java/seedu/moneygowhere/logic/commands/ExchangeRateCommand.java [hunk 2]
```diff
@@ -23,10 +25,11 @@
     public static final String MESSAGE_INVALID_CURRENCY = "Specified currency does not exist";
     public static final String MESSAGE_SAME_CURRENCY = "Specified currency SGD will return the same result.";
 
-
     private double amount;
     private String currencyCode;
 
+    private final Logger logger = Logger.getLogger("Exchange Rate Command Logger");
```
Comment (jonathantjendana) [inline src/main/java/seedu/moneygowhere/logic/commands/ExchangeRateCommand.java]: Should use Logger.getLogger(ExchangeRateCommand.class) instead

File: src/main/java/seedu/moneygowhere/logic/commands/CurrencyCommand.java [hunk 3]
```diff
@@ -27,6 +28,8 @@
 
     private String currency;
 
+    private final Logger logger = Logger.getLogger("Currency Command Logger");
```
Comment (jonathantjendana) [inline src/main/java/seedu/moneygowhere/logic/commands/CurrencyCommand.java]: Should use Logger.getLogger(CurrencyCommand.class)

File: src/main/java/seedu/moneygowhere/logic/commands/SortCommand.java [hunk 4]
```diff
@@ -32,6 +34,8 @@
 
     private final LinkedHashSet<SortField> fields;
 
+    private final Logger logger = Logger.getLogger("Sort Command Logger");
```
Comment (jonathantjendana) [inline src/main/java/seedu/moneygowhere/logic/commands/SortCommand.java]: Should use Logger.getLogger(SortCommand.class) instead

**Review by jonathantjendana — CHANGES_REQUESTED** [review]

**Review by choongyx — APPROVED** [review]

**Review by austinsantoso — APPROVED** [review]
lgtm

**Review by jonathantjendana — APPROVED** [review]
Good job!!

**Pull request merged**